src/jit/unwindarm.cpp

```diff
@@ -415,6 +712,98 @@ void Compiler::unwindEmitFunc(FuncInfoDsc* func, void* pHotCode, void* pColdCode
     static_assert_no_msg(FUNC_HANDLER == (FuncKind)CORJIT_FUNC_HANDLER);
     static_assert_no_msg(FUNC_FILTER == (FuncKind)CORJIT_FUNC_FILTER);
 
+#if defined(_TARGET_UNIX_)
+    if (generateCFIUnwindCodes())
+    {
```

Comment:
Can you extract this out to a `unwindEmitFuncCFI`?

Reply:
@BruceForstall Done